packages/default/scss/orgchart/_layout.scss

```diff
@@ -0,0 +1,92 @@
+@include exports("orgchart/layout") {
+
+    // OrgChart
+    .k-orgchart {
+        width: 100%;
+        box-sizing: border-box;
+        font-size: $orgchart-font-size;
+        font-family: $orgchart-font-family;
+        line-height: $orgchart-line-height;
+        display: flex;
+        justify-content: center;
+        overflow: auto;
```

Comment:
if everything is in the orgchart container and everything is positioned, there is no real need for the element root element to be flex (block is fine) or have overflow.

Reply:
The container width and height are dynamically calculated by the component and is centered so that expanding/collapsing nodes does not create a 'jump effect'.

The overflow is needed for creating scroll when the chart is expanded past the set width.

Reply:
Flex styles removed from `k-orgchart` as they were indeed preventing the `k-orgchart-container` element to expand as needed.